src/repl/scala/tools/nsc/interpreter/PresentationCompilation.scala

```diff
@@ -89,7 +89,7 @@ trait PresentationCompilation { self: IMain =>
 
   private var lastCommonPrefixCompletion: Option[String] = None
 
-  abstract class PresentationCompileResult(val compiler: interactive.Global, val inputRange: Position, val cursor: Int, val buf: String) extends PresentationCompilationResult {
+  private abstract class PresentationCompileResult(val compiler: interactive.Global, val inputRange: Position, val cursor: Int, val buf: String) extends PresentationCompilationResult {
```

Comment:
I intended this to be usable by tools build on top of the REPL (web UIs for instance) that need to be tightly coupled to compiler internals but want the convenience of this API. So I'd suggest to leave it public.